Code/GraphMol/RGroupDecomposition/RGroupCore.h

```diff
@@ -60,88 +60,95 @@ struct RCore {
   }
 
   // Return a copy of core where dummy atoms are replaced by
-  // the atomic number of the respective matching atom in mol
-  ROMOL_SPTR replaceCoreDummiesWithMolMatches(
-      bool &hasDummies, const ROMol &mol, const MatchVectType &match) const {
-    auto coreReplacedDummies = boost::make_shared<RWMol>(*core);
-    hasDummies = false;
+  // the respective matching atom in mol, while other atoms have
+  // their aromatic flag and formal charge copied from from
+  // the respective matching atom in mol
+  ROMOL_SPTR replaceCoreAtomsWithMolMatches(bool &hasCoreDummies,
+                                            const ROMol &mol,
+                                            const MatchVectType &match) const {
+    auto coreReplacedAtoms = boost::make_shared<RWMol>(*core);
+    hasCoreDummies = false;
     for (const auto &p : match) {
-      auto atom = coreReplacedDummies->getAtomWithIdx(p.first);
-      if (isAnyAtomWithMultipleNeighborsOrNotUserRLabel(*atom)) {
-        hasDummies = true;
+      auto atom = coreReplacedAtoms->getAtomWithIdx(p.first);
+      hasCoreDummies |= (atom->getAtomicNum() == 0);
```

Comment:
I think it's more readable this way:
```suggestion
      if(atom->getAtomicNum() == 0){
        hasCoreDummies = true;
      }
```

Reply:
No probs, done. I was 99.9% sure that wouldn't pass the `greg-clang-format` filter, but I thought I'd try. I think it's a good thing also to check your attention level and keep you snappy.

Reply:
> No probs, done. I was 99.9% sure that wouldn't pass the `greg-clang-format` filter, but I thought I'd try. I think it's a good thing also to check your attention level and keep you snappy.

Thanks for giving me that opportunity! ;-)